src/main/java/net/coderbot/iris/uniforms/Uniforms.java

```diff
@@ -62,6 +74,19 @@ public void update() {
 
 
 		updateVector(cameraPosition, MinecraftClient.getInstance().gameRenderer.getCamera().getPos());
+		
+		GL21.glUniform1f(eyeAltitude, (float)MinecraftClient.getInstance().player.getPos().getY());
+		
+		// TODO: Simplify 
+		// Help is it player or cameraEntity
+		int EyeInWater = 0;
```

Comment:
java naming conventions
```suggestion
		int eyeInWater = 0;
```